onnxruntime/python/tools/transformers/fusion_rotary_attention.py

```diff
@@ -445,6 +515,7 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
 
         # k_nodes_1 is for LLaMA-2 Microsoft
         # k_nodes_2 is for LLaMA-2 Hugging Face
+        # k_nodes_4 is for distributed LLaMA-2 Hugging Face
```

Comment:
Same comment to rename it to `LLaMA-2 70B model`

Reply:
Yes, here I should put 70B model, updated